aptos-move/block-executor/src/executor.rs

```diff
@@ -248,6 +246,8 @@ where
             .read_set(idx_to_validate)
             .expect("[BlockSTM]: Prior read-set must be recorded");
 
+        // Note: we validate delayed field reads only at try_commit.
```

Comment:
Let's add a TODO here so later we can validate it, but if it fails, we mark it as a soft failure, requires some new statuses in the scheduler (i.e. not re-execute unless some other part of the validation fails or until commit, but mark as estimates).